src/main/java/bio/overture/songsearch/graphql/GraphQLProvider.java

```diff
@@ -130,6 +130,10 @@ private RuntimeWiring buildWiring() {
         .type(
             newTypeWiring("Query")
                 .dataFetcher("files", fileDataFetcher.getFilesDataFetcher()))
+        .type(
+            newTypeWiring("Query")
+                .dataFetcher(
+                    "analysisAndSampleMatchedAnalysis", analysisDataFetcher.getAnalysisAndSampleMatchedAnalysesFetcher()))
```

Comment:
I think we can just call this query "matchedAnalysisPair", it matches the model like other query/models in this repo and is I think more intuitive then the current name. Feel free to get additional opinions on this from Rosi and Linda though since they are the actual users of this in the end.

Reply:
:thinking: hmm good point I'll get some second opinions on this query

Reply:
as per our discussion I've updated the query to return array of normalSample and tumourSample analysis pairs.. also I set the names so they follow our model/query structure